gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -127,10 +129,11 @@ def run(argv=None):
                                              known_args.output_file)
 
 
-def _write_vcf_meta_info(representative_header_file):
-  # TODO(allieychen): infer the meta information from the schema and replace the
-  # empty `VcfHeader` below.
-  header_fields = vcf_header_io.VcfHeader()
+def _write_vcf_meta_info(input_table, representative_header_file):
+  # type: (str, str) -> None
+  """Writes the meta information generated from BigQuery schema."""
+  header_fields = (bigquery_vcf_schema_converter.
```

Comment:
nit: please try to avoid breaking on the `.` as it looks like it's a tuple of multiple objects rather than one object....does this work?
```
  header_fields = (
      bigquery_vcf_schema_converter. generate_header_fields_from_schema(
          _get_schema(input_table)))
```

Reply:
Thank you Asha!